pkg/resource/table/hooks.go

```diff
@@ -150,6 +163,45 @@ func (rm *resourceManager) customUpdateTable(
 	return &resource{ko}, nil
 }
 
+// syncTTL updates a dynamodb table's TimeToLive property.
+func (rm *resourceManager) syncTTL(
+	ctx context.Context,
+	latest *resource,
+	desired *resource,
```

Comment:
Please place `desired` before `latest` to align with the order that those parameters are supplied to both `sdkUpdate` and `customUpdateTable`